ydb/library/yql/udfs/common/knn/knn-serializer.h

```diff
@@ -129,11 +143,11 @@ class TKnnBitVectorSerializer {
         }
     }
 
-    static const TArrayRef<const ui64> GetArray64(const TStringRef& str) {
+    static std::pair<const ui64*, ui64> GetArray(TStringRef str) {
         const char* buf = str.Data();
-        const size_t len = (str.Size() - HeaderLen) / sizeof(ui64);
-
-        return MakeArrayRef(reinterpret_cast<const ui64*>(buf), len);
+        Y_ASSERT(str.Size() >= 2);
+        const ui64 len = 8 * (str.Size() - HeaderLen - 1) + buf[str.Size() - HeaderLen - 1];
```

Comment:
This should be explained in comments just before TKnnBitVectorSerializer

Reply:
done